### PR TITLE
CAMEL-11894: camel-karaf-commands deployment failed on karaf 4.0

### PR DESCRIPTION
**Pull request by frouleau**
Fix OSGi's version range for the missing package

### CODE REVIEW

File: platforms/karaf/commands/pom.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
           <instructions>
             <Import-Package>
               !org.apache.felix.utils.properties,
-              org.apache.karaf.shell.api.action;version="[4,5)",
+              org.apache.karaf.shell.api.*;version="[4,5)",
               org.apache.karaf.shell.support.completers;version="[4,5)",
               org.slf4j.*;version="[1.6,2)",
```
Comment (gnodet) [inline platforms/karaf/commands/pom.xml]: Can you conflate both lines into the following ?
```
             org.apache.karaf.shell.*;version="[4,5)",
```

Comment (frouleau) [reply platforms/karaf/commands/pom.xml]: Sure, will be better !

**Comment by oscerd** [comment]
I'm merging this.

**Comment by oscerd** [comment]
Thanks @frouleau the PR has been merged. Can you close this?

**Pull request closed without merging**